workers/api-server/src/router.rs

```diff
@@ -33,27 +33,16 @@ const ERR_INVALID_QUERY_PARAMS: &str = "invalid query params";
 // | Helpers |
 // -----------
 
-/// Builds an empty HTTP 400 (Bad Request) response
-pub(super) fn build_400_response(err: String) -> Response<Body> {
-    Response::builder().status(StatusCode::BAD_REQUEST).body(Body::from(err)).unwrap()
-}
-
-/// Builds an empty HTTP 404 (Not Found) response
-pub(super) fn build_404_response(err: String) -> Response<Body> {
-    Response::builder().status(StatusCode::NOT_FOUND).body(Body::from(err)).unwrap()
-}
-
-/// Builds an empty HTTP 500 (Internal Server Error) response
-pub(super) fn build_500_response(err: String) -> Response<Body> {
-    Response::builder().status(StatusCode::INTERNAL_SERVER_ERROR).body(Body::from(err)).unwrap()
-}
```

Comment:
I would still use these methods to be clear, they might be useful when building other routes that return 404, 400, etc. So I wouldn't get rid of them, maybe just change their implementation to call out to `build_response_from_status_code`